### PR TITLE
Add SELinux instructions and FAQ

### PR DESCRIPTION
**Pull request by javiertury**
Create a FAQ section and document how to build a SELinux policy package. [Related issue](https://github.com/pgroonga/pgroonga/issues/91). 

Perhaps I made some mistakes since I'm not familiar with the documentation builder. Also, these changes are only for the English section. I would be nice if someone that knows Japanese could translate them.

### CODE REVIEW

**Comment by kenhys** [comment]
Great! LGTM :+1:

File: install/source.md [hunk 1]
```diff
@@ -73,6 +73,29 @@ Install PGroonga:
...
+require {
+	type postgresql_t;
+	type postgresql_db_t;
+	class file map;
+}
+
+allow postgresql_t postgresql_db_t:file map;
+EOF
+
+% checkmodule -M -m -o my-pgroonga.mod my-pgroonga.te
```
Comment (komainu8) [inline install/source.md]: We should write need to install checkpolicy package to use checkmodule command.

Comment (javiertury) [reply install/source.md]: Totally, I forgot about dependencies.

**Comment by javiertury** [comment]
I have squashed the commits and force-pushed the branch

**Comment by kou** [comment]
Thanks!

**Pull request merged**